.pipelines/SecretManagement-Official.yml

```diff
@@ -0,0 +1,189 @@
+  template: v2/OneBranch.Official.CrossPlat.yml@templates
+  parameters:
+    globalSdl: # https://aka.ms/obpipelines/sdl
+      asyncSdl:
+        enabled: true
+        forStages: [build]
+    featureFlags:
+      EnableCDPxPAT: false
+      WindowsHostVersion:
+        Version: 2022
+        Network: Netlock
+    stages:
+    - stage: build
+      jobs:
+      - job: main
+        displayName: Build package
+        pool:
+          type: windows
+        variables:
+          ob_outputDirectory: $(Build.SourcesDirectory)/out
+        steps:
+          - pwsh: |
+              [xml]$xml = Get-Content Directory.Build.props
+              $version = $xml.Project.PropertyGroup.ModuleVersion
+              Write-Output "##vso[task.setvariable variable=version;isOutput=true]$version"
+            name: package
+            displayName: Get version from project properties
+          - task: onebranch.pipeline.version@1
+            displayName: Set OneBranch version
+            inputs:
+              system: Custom
+              customVersion: $(package.version)
+          - task: UseDotNet@2
+            displayName: Use .NET SDK
+            inputs:
+              packageType: sdk
+              useGlobalJson: true
+          - pwsh: |
+              Register-PSRepository -Name CFS -SourceLocation "https://pkgs.dev.azure.com/powershell/PowerShell/_packaging/powershell/nuget/v2" -InstallationPolicy Trusted
+              Install-Module -Repository CFS -Name Microsoft.PowerShell.PSResourceGet
+              ./tools/installPSResources.ps1 -PSRepository CFS
+            displayName: Install PSResources
+          - pwsh: Invoke-Build -Configuration $(BuildConfiguration) -Task Build, Test
+            displayName: Build
+          - task: onebranch.pipeline.signing@1
+            displayName: Sign 1st-party files in module
+            inputs:
+              command: sign
+              signing_profile: external_distribution
+              search_root: $(Build.SourcesDirectory)/module
+              files_to_sign: |
+                Microsoft.*.dll;
+                Microsoft.*.psd1;
+                Microsoft.*.ps1xml;
+          - task: onebranch.pipeline.signing@1
+            displayName: Sign 1st-party files in library
+            inputs:
+              command: sign
+              signing_profile: external_distribution
+              search_root: $(Build.SourcesDirectory)/artifacts
+              files_to_sign: |
+                publish/**/Microsoft.PowerShell.SecretManagement.dll;
+                refs/**/Microsoft.PowerShell.SecretManagement.dll;
+          - task: ArchiveFiles@2
+            displayName: Zip module
+            inputs:
+              rootFolderOrFile: $(Build.SourcesDirectory)/module
+              includeRootFolder: false
+              archiveType: zip
+              archiveFile: out/SecretManagement-v$(package.version).zip
+          - pwsh: Invoke-Build -Configuration $(BuildConfiguration) Package
```

Comment:
should this be -Package (or should a named parameter be present?)

Reply:
Nope, it's the name of an InvokeBuild task so it's an argument, not a parameter.

Reply:
(The default argument, which could be specified as `-Task Package`.)